scripts/repos/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Is this package.json is meant to be here? Same for the yarn.lock in the same dir